test/cf-expandables.html

```diff
@@ -158,9 +158,9 @@
 
     <!-- </div> -->
 
-    <script src="../src/vendor/jquery/jquery.js"></script>
-    <script src="../src/vendor/jquery.easing/jquery.easing.js"></script>
-    <script src="../src/vendor/qunit/qunit.js"></script>
+    <script src="../src/vendor/jquery/dist/jquery.js"></script>
```

Comment:
It's probably fine but `jquery.min.js` would be the production environment while `jquery.js` would be the dev environment, so in case there was some kind of crazy edge case where minification created an issue with cf-expandables it would show up here.